TCAT/WhatsNewHeaderView.swift

```diff
@@ -0,0 +1,155 @@
+    func okButtonPressed()
+    func cardPressed()
+}
+
+class WhatsNewHeaderView: UIView {
+
+    var whatsNewDelegate: WhatsNewDelegate?
+    var updateTitle: UILabel!
+    var dismissButton: UIButton!
+    var updateDescription: UILabel!
+    var whatsNewHeader: UILabel!
+    var backgroundView: UIView!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        createBackgroundView()
+        createUpdateDescription()
+        createWhatsNewHeader()
+        createUpdateTitle()
+        createDismissButton()
+        updateConstraints()
+    }
+    
+    func createBackgroundView() {
+        backgroundView = UIView()
+        backgroundView.backgroundColor = .white
+        backgroundView.layer.cornerRadius = 14
+        backgroundView.clipsToBounds = true
+        backgroundView.layer.borderWidth = 1
+        backgroundView.layer.borderColor = UIColor(red: 233/255, green: 235/255, blue: 238/255, alpha: 1).cgColor
+        
+        let wholeCardTapped = UITapGestureRecognizer()
+        wholeCardTapped.addTarget(self, action: #selector(cardTapped))
+        backgroundView.addGestureRecognizer(wholeCardTapped)
+        
+        addSubview(backgroundView)
+    }
+    
+    func createWhatsNewHeader() {
+        whatsNewHeader = UILabel()
+        whatsNewHeader.text = "NEW IN ITHACA TRANSIT"
+        whatsNewHeader.font = UIFont(name: Constants.Fonts.SanFrancisco.Semibold, size: 10)
+        whatsNewHeader.textColor = UIColor(red: 8/255, green: 160/255, blue: 224/255, alpha: 1)
+        
+        backgroundView.addSubview(whatsNewHeader)
+    }
+    
+    func createUpdateTitle() {
+        updateTitle = UILabel()
+        updateTitle.text = "App Shortcuts for Favorites"
+        updateTitle.font = UIFont(name: Constants.Fonts.SanFrancisco.Bold, size: 16)
+        
+        backgroundView.addSubview(updateTitle)
+    }
+    
+    
+    func createUpdateDescription () {
+        updateDescription = UILabel()
+        updateDescription.text = "Force Touch the app icon to search your favorites even faster."
+        updateDescription.font = UIFont(name: Constants.Fonts.SanFrancisco.Regular, size: 14)
+        updateDescription.textColor = UIColor(red: 144/255, green: 148/255, blue: 156/255, alpha: 1)
+        updateDescription.numberOfLines = 0
+        updateDescription.textAlignment = .center
+        
+        backgroundView.addSubview(updateDescription)
+    }
+    
+    func createDismissButton() {
+        dismissButton = UIButton()
+        dismissButton.setTitle("OK", for: .normal)
+        dismissButton.titleLabel?.font = UIFont(name: Constants.Fonts.SanFrancisco.Semibold, size: 14)
+        dismissButton.addTarget(self, action: #selector(okButtonPressed), for: .touchUpInside)
+        dismissButton.backgroundColor = UIColor(red: 8/255, green: 160/255, blue: 224/255, alpha: 1)
+        dismissButton.setTitleColor(.white, for: .normal)
+        dismissButton.layer.cornerRadius = 14.5
```

Comment:
I know you're just pulling from Zeplin, but I'd just make this 16 for now, this is a wonky number.